transcom/mymove
# B 21336 main Add bulk assignment button and modal creation

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21336)

## Summary
Create new feature flag called "FEATURE_FLAG_BULK_ASSIGNMENT."
Add a feature flagged button to the SC/TOO/TIO queues, only visible to supervisors.
Clicking the button will open a Bulk Assignment modal with just the Title, Save and Cancel button.

### How to test
1. With the FEATURE_FLAG_BULK_ASSIGNMENT OFF Check the SC/TOO/TIO and verify the bulk assignment Button is not visible
2. Turn the FEATURE_FLAG_BULK_ASSIGNMENT ON and create a TOO/SC/TIO all without supervisor privileges 
3. check the SC/TOO/TIO queues and verify the bulk assignment button is not visible
4. create a TOO/SC/TIO all with supervisor privileges
5. check the SC/TOO/TIO queues and verify the bulk assignment button is visible
6. Click the bulk assignment button and verify the bulk assignment modal pops up with a title save and cancel button

[INT PR](https://github.com/transcom/mymove/pull/14457)

[FeatureFlag PR](https://github.com/transcom/milmove-feature-flags/pull/241)
## Screenshots
